### PR TITLE
[metric learning] fix windows download dataset

### PR DESCRIPTION
**Pull request by kebinC**

### CODE REVIEW

File: PaddleCV/metric_learning/README.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ cd data/
...
+├──Stanford_Online_Products
+│   ├── bicycle_final
+│   ├── bicycle_final.txt
+│   ├── cabinet_final
+│   ├── cabinet_final.txt
+|   ...
+├──Stanford_Online_Products.zip 
+```
+
+if you are windows user, you can download the dataset using below ftp link and unzip file to ```data``` directory.
```
Comment (qingqing01) [inline PaddleCV/metric_learning/README.md]: windows -> Windows

Comment (qingqing01) [inline PaddleCV/metric_learning/README.md]: windows -> Windows

Comment (kebinC) [reply PaddleCV/metric_learning/README.md]: ok

Comment (kebinC) [reply PaddleCV/metric_learning/README.md]: ok

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**